context/.swcrc

```diff
@@ -11,6 +11,7 @@
             "decorators": false,
             "decoratorsBeforeExport": false
         },
+        "externalHelpers": true,
```

Comment:
Do we need to install `@swc/helpers` as https://swc.rs/docs/configuration/compilation#jscexternalhelpers indicates?

Reply:
Yes, this was installed in the other branch once I realized it was missing: https://github.com/hubmapconsortium/portal-ui/blob/02a4743766c2035a503482f327b5cfd47fe2c0f0/context/package.json#L89

This ended up being necessary to get tests to run successfully.